src/scripts/cognac_chatbot.js

```diff
@@ -0,0 +1,20 @@
+const kolmafia = require("kolmafia");
+
+module.exports.main = function main(sender, message, channel) {
+  let stench = kolmafia.getProperty("_cognac_heapStench");
+  if (channel == "/hobopolis" && sender == "Dungeon") {
+    if (message.endsWith("moved some trash out of the Purple Light District.") && stench != "") {
+      stench = parseInt(stench) + 1;
+      kolmafia.setProperty("_cognac_heapStench", stench);
+      kolmafia.chatClan(`Stench level changed to ${stench}`, "hobopolis");
```

Comment:
Can we get away with only emitting this event some of the time? Say proportional to the number of players? I believe there's a pref that already tracks that value.

Reply:
The debugger in me likes more data and it is interesting to see it change. I can def see it becoming just a blur of text though when we have many folks running this code. I'll update it to only emit when stench is changed to 25, 50, 75, 100% or above 100% of target

Reply:
the main thing is each participant is writing this message, for every instance of stench increase, which itself increases with the number of players. 

this is just a nitpick though, can certainly iterate if ever this is too chatty.

Reply:
If there are 15 people diving, 3 requesting compost, that line will write in chat 26*15=390 times per dive.

Reply:
> The debugger in me likes more data and it is interesting to see it change. I can def see it becoming just a blur of text though when we have many folks running this code. I'll update it to only emit when stench is changed to 25, 50, 75, 100% or above 100% of target

Implemented this approach. So far so good. Wanted to print everything above goal as its interesting to see overshoot